src/lib/executeSourceTransfer.js

```diff
@@ -2,6 +2,12 @@
 
 const log = require('../common').log.create('executeSourceTransfer')
 const validator = require('./validate')
+const promiseRetry = require('promise-retry')
+const retryOpts = { // retry 10 times within 5 seconds
+  retries: 10,
+  factor: 1,
+  minTimeout: 500
+}
```

Comment:
When ledger and connector communicate over a network, the round trip time is likely to exceed `10ms`. If we set the retry to start after `10ms`, each fulfillment is likely to be retried because the ledger's response has not yet arrived. How about `50ms`?